server/course/index.js

```diff
@@ -1,27 +1,52 @@
 'use strict';
 
+const { NOT_FOUND, UNAUTHORIZED } = require('http-status-codes');
 const { authorize } = require('../shared/auth/mw');
 const { Course } = require('../shared/database');
 const { createError } = require('../shared/error/helpers');
-const { NOT_FOUND, UNAUTHORIZED } = require('http-status-codes');
 const ctrl = require('./course.controller');
+const path = require('path');
 const processQuery = require('../shared/util/processListQuery')();
 const router = require('express-promise-router')();
 
+const activity = require('../activity');
+const comment = require('../comment');
+const revision = require('../revision');
+const teachingElement = require('../teaching-element');
+
+module.exports = { path: '/courses', router };
+
+router
+  .use(hasAccess)
+  .param('id', getCourse);
+
+router
+  .route('/')
+  .get(processQuery, ctrl.index)
+  .post(authorize(), ctrl.create);
+
+router
+  .route('/:id')
+  .get(ctrl.get)
+  .patch(ctrl.patch)
+  .delete(ctrl.remove);
+
 router
-  .param('id', getCourse)
-  .use('/courses/:id*', hasAccess)
-  .get('/courses', processQuery, ctrl.index)
-  .post('/courses', authorize(), ctrl.create)
-  .get('/courses/:id', ctrl.get)
-  .patch('/courses/:id', ctrl.patch)
-  .delete('/courses/:id', ctrl.remove)
-  .post('/courses/:id/clone', authorize(), ctrl.clone)
-  .post('/courses/:id/publish', ctrl.publishRepoInfo)
-  .get('/courses/:id/users', ctrl.getUsers)
-  .post('/courses/:id/users', ctrl.upsertUser)
-  .delete('/courses/:id/users/:userId', ctrl.removeUser)
-  .get('/courses/:id/contentInventory', ctrl.exportContentInventory);
+  .post('/:id/clone', authorize(), ctrl.clone)
+  .post('/:id/publish', ctrl.publishRepoInfo)
+  .get('/:id/users', ctrl.getUsers)
+  .post('/:id/users', ctrl.upsertUser)
+  .delete('/:id/users/:userId', ctrl.removeUser)
+  .get('/:id/contentInventory', ctrl.exportContentInventory);
```

Comment:
Rename the route to use a [hyphen](https://restfulapi.net/resource-naming/) for multiword names instead of camel case.